src/Mvc/test/WebSites/FilesWebSite/Controllers/DownloadFilesController.cs

```diff
@@ -44,6 +44,16 @@ public IActionResult DownloadFromDiskWithFileName_WithLastModifiedAndEtag()
         return PhysicalFile(path, "text/plain", "downloadName.txt", lastModified, entityTag, true);
     }
 
+    public IActionResult DownloadFromDiskSymlink()
+    {
+        var path = Path.Combine(_hostingEnvironment.ContentRootPath, "sample.txt");
+        var symlink = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
```

Comment:
This is ok, but it might be nice to do a cleanup step at the end of the test. E.g.

```C#
var symLink = Path.Combine(Path.GetTempPath(), "mvctests", Path.GetRandomFileName());
Directory.CreateDirectory(Path.GetDirectoryName(symLink));

...

FileResultTests : IDisposable
{
   public void Dispose()
   {
       try { Directory.Delete(Path.Combine(Path.GetTempPath(), "mvctests"), recursive: true); } catch {}
    }
}